_posts/2020-09-04-issue-10.md

```diff
@@ -34,7 +34,7 @@ At almost 40, I realize that majority of us tech workers look nothing like what
 
 Now, coming up on my eighth year working for startups, what do I have to show for my pockmarked resume at a string of startups that have faded into obscurity? Beyond cynicism for the tech worker social contract and a network of equally cynical peers, I finally landed a six-figure salary. My current company is doing well; it’s even possible that we’ll cross the valley of hypergrowth to an IPO. Still, repeated experiences of watching leadership teams commit to unachievable goals and failing dismally has me looking over my shoulder. 
 
-I don’t think it’s asking too much that I earn a decent salary, have job stability, keep reasonable hours, have autonomy in my work, and feel that I can shape the direction and values of my company. Startup employees are often lured in with the promise that they’ll own stock in the company, but owning 0.001% of a company when the CEO owns 30% is a far cry from co-ownership. In practice, startups are rarely the egalitarian dream their founders present them to be. Many tech workers, myself included, would rather be baking bread.
+I don’t think it’s asking too much that I earn a decent salary, have job stability, keep reasonable hours, have autonomy in my work, and feel that I can shape the direction and values of my company. Startup employees are often lured in with the promise that they’ll own stock in the company, but owning 0.05% of a company when the CEO owns 30% is a far cry from co-ownership. In practice, startups are rarely the egalitarian dream their founders present them to be. Many tech workers, myself included, would rather be baking bread.
```

Comment:
i don't think this is accurate. people really do get `.001` percent of a company. that wasn't an exaggeration.